docs/examples/example.json

```diff
@@ -26,6 +26,13 @@
         "transport",
         "transport.road"
       ],
+      "additionalClassifications": [
+        {
+          "scheme": "COFOG",
```

Comment:
But see [this issue](https://github.com/open-contracting/ocds-extensions/issues/135#issue-601217936): "I think we also agreed in OC4IDS that COFOG is not appropriate for sector classifications"

Reply:
I think COFOG fits the description of `additionalClassifications`, so this is OK:

> One or more additional project classifications can be provided to describe the social or economic focus of the project. This classification may take place against a locally developed codelist, or a globally established codelist.